test_scripts/Capabilities/PersistingHMICapabilities/008_HMI_sends_GetSystemInfo_with_another_ccpu_version.lua

```diff
@@ -0,0 +1,56 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal:https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0249-Persisting-HMI-Capabilities-specific-to-headunit.md
+--
+-- Description: Check that SDL is compare saved ccpu_version parameter and received from HMI.
+--  SDL does requested all capabilities from HMI in case ccpu_version do not match
+--
+-- Preconditions:
+-- 1  Value of HMICapabilitiesCacheFile parameter is defined (hmi_capabilities_cache.json) in smartDeviceLink.ini file
+-- 2. HMI capabilities cache file (hmi_capabilities_cache.json) doesn't exist on file system
+-- 3. SDL and HMI are started
+-- 4. HMI sends all HMI capabilities (VR/TTS/RC/UI etc) to SDL
+-- 5. SDL stored capabilities to HMI capabilities cache file (hmi_capabilities_cache.json) in AppStorageFolder
+-- 6. Ignition OFF/ON cycle performed
+-- Sequence:
+-- 1. HMI sends "BasicCommunication.GetSystemInfo" response with the different ccpu_version
+--  a. SDL sends all HMI capabilities request (VR/TTS/RC/UI etc) to HMI
+-- 2. Ignition OFF/ON cycle performed
+-- 3. HMI sends "BasicCommunication.GetSystemInfo" response with the same ccpu_version
+--  a. SDL does not send HMI capabilities request (VR/TTS/RC/UI etc) to HMI
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local common = require('test_scripts/Capabilities/PersistingHMICapabilities/common')
+
+--[[ Local Functions ]]
+local function getHMIParamsWithOutRequests(pVersion)
+  local hmiCapabilities = common.updateHMISystemInfo(pVersion)
+  hmiCapabilities.RC.GetCapabilities.occurrence = 0
```

Comment:
I see this `occurrence = 0` code a lot, would it make sense to add an optional param to `common.getHMIParamsWithOutRequests` so this code can exist in just one place?

Reply:
@iCollin Updated (see f54d450 )